### PR TITLE
add v0.9.0 docs to site

### PR DESCRIPTION
**Pull request by Bradamant3**
Signed-off-by: JENNIFER RONDEAU <jrondeau@heptio.com>

PTAL staged site: https://bradamant3.github.io/ark/v0.9.0/

### CODE REVIEW

File: master/about.md~ba18ad8... add v0.9.0 docs to site [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# How Ark Works
```
Comment (skriss) [inline master/about.md~ba18ad8... add v0.9.0 docs to site]: This looks like a garbage file (git leftover from something or other), should be removed

File: master/cli-reference/ark.md~ba18ad8... add v0.9.0 docs to site [hunk 2]
```diff
@@ -0,0 +1,46 @@
+## ark
```
Comment (skriss) [inline master/cli-reference/ark.md~ba18ad8... add v0.9.0 docs to site]: This looks like a garbage file (git leftover from something or other), should be removed

File: master/cli-reference/ark_backup_describe.md~ba18ad8... add v0.9.0 docs to site [hunk 3]
```diff
@@ -0,0 +1,39 @@
+## ark backup describe
```
Comment (skriss) [inline master/cli-reference/ark_backup_describe.md~ba18ad8... add v0.9.0 docs to site]: This looks like a garbage file (git leftover from something or other), should be removed

File: master/cli-reference/ark_describe_backups.md~ba18ad8... add v0.9.0 docs to site [hunk 4]
```diff
@@ -0,0 +1,39 @@
+## ark describe backups
```
Comment (skriss) [inline master/cli-reference/ark_describe_backups.md~ba18ad8... add v0.9.0 docs to site]: This looks like a garbage file (git leftover from something or other), should be removed

File: master/cli-reference/ark_describe_restores.md~ba18ad8... add v0.9.0 docs to site [hunk 5]
```diff
@@ -0,0 +1,39 @@
+## ark describe restores
```
Comment (skriss) [inline master/cli-reference/ark_describe_restores.md~ba18ad8... add v0.9.0 docs to site]: This looks like a garbage file (git leftover from something or other), should be removed

File: master/cli-reference/ark_restore_describe.md~ba18ad8... add v0.9.0 docs to site [hunk 6]
```diff
@@ -0,0 +1,39 @@
+## ark restore describe
```
Comment (skriss) [inline master/cli-reference/ark_restore_describe.md~ba18ad8... add v0.9.0 docs to site]: This looks like a garbage file (git leftover from something or other), should be removed

File: master/cli-reference/ark_server.md~ba18ad8... add v0.9.0 docs to site [hunk 7]
```diff
@@ -0,0 +1,40 @@
+## ark server
```
Comment (skriss) [inline master/cli-reference/ark_server.md~ba18ad8... add v0.9.0 docs to site]: This looks like a garbage file (git leftover from something or other), should be removed

File: master/cloud-common.md [hunk 8]
```diff
@@ -77,3 +79,4 @@ After you set up the Ark server, try these examples:
 [3]: namespace.md
 [4]: ibm-config.md
 [19]: https://kubernetes.io/docs/concepts/storage/persistent-volumes/#reclaiming
+[20]: https://github.com/heptio/ark/blob/master/docs/restic.md
```
Comment (skriss) [inline master/cloud-common.md]: This is my bad, but this link should be changed (can do it in a separate PR on master)

File: master/faq.md~ba18ad8... add v0.9.0 docs to site [hunk 9]
```diff
@@ -0,0 +1,38 @@
+# FAQ
```
Comment (skriss) [inline master/faq.md~ba18ad8... add v0.9.0 docs to site]: This looks like a garbage file (git leftover from something or other), should be removed

File: v0.9.0/about.md [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+## On-demand backups
+
+The **backup** operation:
+
+1. Uploads a tarball of copied Kubernetes objects into cloud object storage.
+
+1. Calls the cloud provider API to make disk snapshots of persistent volumes, if specified.
+
+You can optionally specify hooks to be executed during the backup. For example, you might
+need to tell a database to flush its in-memory buffers to disk before taking a snapshot. [More about hooks][10].
```
Comment (skriss) [inline v0.9.0/about.md]: link [10] is missing

**Comment by Bradamant3** [comment]
oy thanks Steve. On it, trying to figure out better now ...

**Comment by Bradamant3** [comment]
removed garbage files, but also fixed links; still need to fix up in master, but that can go separately (with the others ...)

**Comment by skriss** [comment]
A squash/fixup would be great.

**Comment by skriss** [comment]
@nrb @ncdc any comments?

File: master/image-tagging.md [hunk 11]
```diff
@@ -0,0 +1,21 @@
+# Image tagging policy
+
+This document describes Ark's image tagging policy.
+
+## Released versions
+
+`gcr.io/heptio-images/ark:<SemVer>`
+
+Ark follows the [Semantic Versioning](http://semver.org/) standard for releases. Each tag in the `github.com/heptio/ark` repository has a matching image, e.g. `gcr.io/heptio-images/ark:v0.8.0`.
```
Comment (nrb) [inline master/image-tagging.md]: Do we care if this is kept at v0.8.0?

Comment (skriss) [reply master/image-tagging.md]: I think it's fine, it's just an example

File: v0.9.0/debugging-deletes.md [hunk 12]
```diff
@@ -0,0 +1,47 @@
+# Ark version 0.7.0 and later: issue with deleting namespaces and backups
```
Comment (nrb) [inline v0.9.0/debugging-deletes.md]: @ncdc Do we want to keep this section around? afaik it's not relevant in 0.9.0 since the finalizers get stripped.

Comment (ncdc) [reply v0.9.0/debugging-deletes.md]: I'm fine removing it from 0.9+.
